### DISCUSSION TITLE
No Tablet Detected

### DISCUSSION THREAD

**Discussion by Old-Greggu** [OpenTabletDriver/OpenTabletDriver#2431] [Q&A] [answered]
console was showing as follows:

[WinUSB:Info]	Cannot create device for '\\?\usb#vid_256c&pid_006d&mi_00#6&1cbf020e&1&0000#{dee824ef-729b-4a0e-9c14-b7117d33a817}'
[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.

but now its just saying no tablet detected when I reconnect tablet (Gaomon S620), and I have used Zadig's WinUSB for the drivers on both Interfaces that were showing up for the tablet.

Any help would be greatly appreciated, was trying to get my tablet to register on Osu!lazer. T_T

**Comment by jamesbt365** [comment, marked as answer]
You do not need winusb for this tablet. Uninstall it with device manager.

**Reply by Old-Greggu** [reply]
Ok so I ended up doing a fresh install of Windows, been meaning to anyhow. (not sure if that would even have an effect)

[WinUSB:Info]	Cannot create device for '\\?\usb#vid_256c&pid_006d&mi_00#6&1cbf020e&0&0000#{dee824ef-729b-4a0e-9c14-b7117d33a817}'
[WinUSB:Info]	Cannot create device for '\\?\usb#vid_256c&pid_006d&mi_00#6&1cbf020e&0&0000#{62f12d4c-3431-4efd-8dd7-8e9aab18d30c}'
[RootHub:Info]	Debounced WinUSBRootHub's DevicesChanged event
[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.

this is what I'm getting when I plug in the hardware with the OpenTabletDriver running, sorry if I'm just missing something obvious, I don't mess with drivers much.

**Reply by Old-Greggu** [reply]
Ayo, messed around for a bit, finally got it.

**Comment by kobozevdenis510-afk** [comment]
[Detect:Info]   Searching for tablets...
[Detect:Debug]  Searching for tablet 'Acepen AP1060'
[Detect:Debug]  Searching for tablet 'Acepen AP906'
[Detect:Debug]  Searching for tablet 'Adesso Cybertablet K8'
[Detect:Debug]  Searching for tablet 'Artisul A1201'
[Detect:Debug]  Searching for tablet 'Artisul AP604'
[Detect:Debug]  Searching for tablet 'Artisul D16 Pro'
[Detect:Debug]  Searching for tablet 'Artisul M0610 Pro'
[Detect:Debug]  Searching for tablet 'FlooGoo FMA100'
[Detect:Debug]  Searching for tablet 'Gaomon 1060 Pro'
[Detect:Debug]  Searching for tablet 'Gaomon GM116HD'
[Detect:Debug]  Searching for tablet 'Gaomon GM156HD'
[Detect:Debug]  Searching for tablet 'Gaomon M106K Pro'
[Detect:Debug]  Searching for tablet 'Gaomon M106K'
[Detect:Debug]  Searching for tablet 'Gaomon M10K Pro'
[Detect:Debug]  Searching for tablet 'Gaomon M10K'
[Detect:Debug]  Searching for tablet 'Gaomon M1220'
[Detect:Debug]  Searching for tablet 'Gaomon M1230'
[Detect:Debug]  Searching for tablet 'Gaomon M6'
[Detect:Debug]  Searching for tablet 'Gaomon M8 (Variant 2)'
[Detect:Debug]  Searching for tablet 'Gaomon M8'
[Detect:Debug]  Searching for tablet 'Gaomon PD1161'
[Detect:Debug]  Searching for tablet 'Gaomon PD156 Pro'
[Detect:Debug]  Searching for tablet 'Gaomon PD1560'
[Detect:Debug]  Searching for tablet 'Gaomon PD1561'
[Detect:Debug]  Searching for tablet 'Gaomon PD2200'
[Detect:Debug]  Searching for tablet 'Gaomon S56K'
[Detect:Debug]  Searching for tablet 'Gaomon S620'
[Detect:Debug]  Searching for tablet 'Gaomon S630'
[Detect:Debug]  Searching for tablet 'Gaomon S830'
[Detect:Debug]  Searching for tablet 'Genius G-Pen 560'
[Detect:Debug]  Searching for tablet 'Genius i405x'
[Detect:Debug]  Searching for tablet 'Genius i608x'
[Detect:Debug]  Searching for tablet 'Huion 1060 Plus'
[Detect:Debug]  Searching for tablet 'Huion 420'
[Detect:Debug]  Searching for tablet 'Huion G10T'
[Detect:Debug]  Searching for tablet 'Huion G930L'
[Detect:Debug]  Searching for tablet 'Huion GC610'
[Detect:Debug]  Searching for tablet 'Huion GT-156HD V2'
[Detect:Debug]  Searching for tablet 'Huion GT-220 V2'
[Detect:Debug]  Searching for tablet 'Huion GT-221 Pro'
[Detect:Debug]  Searching for tablet 'Huion GT-221'
[Detect:Debug]  Searching for tablet 'Huion H1060P'
[Detect:Debug]  Searching for tablet 'Huion H1061P'
[Detect:Debug]  Searching for tablet 'Huion H1161'
[Detect:Debug]  Searching for tablet 'Huion H320M'
[Detect:Debug]  Searching for tablet 'Huion H420'
[Detect:Debug]  Searching for tablet 'Huion H420X'
[Detect:Debug]  Searching for tablet 'Huion H430P'
[Detect:Debug]  Searching for tablet 'Huion H580X'
[Detect:Debug]  Searching for tablet 'Huion H610 Pro V2'
[Detect:Debug]  Searching for tablet 'Huion H610 Pro V3'
[Detect:Debug]  Searching for tablet 'Huion H610 Pro'
[Detect:Debug]  Searching for tablet 'Huion H610X'
[Detect:Debug]  Searching for tablet 'Huion H640P'
[Detect:Debug]  Searching for tablet 'Huion H641P'
[Detect:Debug]  Searching for tablet 'Huion H642'
[Detect:Debug]  Searching for tablet 'Huion H690'
[Detect:Debug]  Searching for tablet 'Huion H950P'
[Detect:Debug]  Searching for tablet 'Huion H951P'
[Detect:Debug]  Searching for tablet 'Huion HC16'
[Detect:Debug]  Searching for tablet 'Huion HS610'
[Detect:Debug]  Searching for tablet 'Huion HS611'
[Detect:Debug]  Searching for tablet 'Huion HS64'
[Detect:Debug]  Searching for tablet 'Huion HS95'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 12'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 13 (Gen 3)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 13'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 16 (2021)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 16'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 20'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 22 Plus'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 22'
[Detect:Debug]  Searching for tablet 'Huion Kamvas 24 Plus'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 12'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 13 (2.5k)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 13'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 16 (2.5k)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 16 (4k)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 16 Plus (4k)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 16'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 19 (4K)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 20'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 22 (2019)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 24 (4K)'
[Detect:Debug]  Searching for tablet 'Huion Kamvas Pro 24'
[Detect:Debug]  Searching for tablet 'Huion New 1060 Plus (2048)'
[Detect:Debug]  Searching for tablet 'Huion New 1060 Plus'
[Detect:Debug]  Searching for tablet 'Huion Q11K V2'
[Detect:Debug]  Searching for tablet 'Huion Q11K'
[Detect:Debug]  Searching for tablet 'Huion Q620M'
[Detect:Debug]  Searching for tablet 'Huion Q630M'
[Detect:Debug]  Searching for tablet 'Huion RDS-160'
[Detect:Debug]  Searching for tablet 'Huion RTE-100'
[Detect:Debug]  Searching for tablet 'Huion RTM-500'
[Detect:Debug]  Searching for tablet 'Huion RTP-700'
[Detect:Debug]  Searching for tablet 'Huion WH1409 V2 (Variant 2)'
[Detect:Debug]  Searching for tablet 'Huion WH1409 V2'
[Detect:Debug]  Searching for tablet 'Huion WH1409'
[Detect:Debug]  Searching for tablet 'KENTING K5540'
[Detect:Debug]  Searching for tablet 'LifeTec LT9570'
[Detect:Debug]  Searching for tablet 'Monoprice 10594'
[Detect:Debug]  Searching for tablet 'Monoprice MP1060-HA60'
[Detect:Debug]  Searching for tablet 'Parblo A609'
[Detect:Debug]  Searching for tablet 'Parblo A610 Pro'
[Detect:Debug]  Searching for tablet 'Parblo A610'
[Detect:Debug]  Searching for tablet 'Parblo A640 V2'
[Detect:Debug]  Searching for tablet 'Parblo A640'
[Detect:Debug]  Searching for tablet 'Parblo Intangbo M'
[Detect:Debug]  Searching for tablet 'Parblo Intangbo S'
[Detect:Debug]  Searching for tablet 'Parblo Ninos M'
[Detect:Debug]  Searching for tablet 'Parblo Ninos N4'
[Detect:Debug]  Searching for tablet 'Parblo Ninos N7'
[Detect:Debug]  Searching for tablet 'Parblo Ninos N7B'
[Detect:Debug]  Searching for tablet 'Parblo Ninos S'
[Detect:Debug]  Searching for tablet 'RobotPen T9A'
[Detect:Debug]  Searching for tablet 'Trust Flex Design Tablet'
[Detect:Debug]  Searching for tablet 'Turcom TS-6580'
[Detect:Debug]  Searching for tablet 'UC-Logic 1060N'
[Detect:Debug]  Searching for tablet 'UC-Logic PF1209'
[Detect:Debug]  Searching for tablet 'UGEE M708 V2'
[Detect:Debug]  Searching for tablet 'UGEE M708'
[Detect:Debug]  Searching for tablet 'UGEE M808'
[Detect:Debug]  Searching for tablet 'UGEE M908'
[Detect:Debug]  Searching for tablet 'UGEE S1060'
[Detect:Debug]  Searching for tablet 'UGEE S640'
[Detect:Debug]  Searching for tablet 'UGEE U1200'
[Detect:Debug]  Searching for tablet 'UGEE U1600'
[Detect:Debug]  Searching for tablet 'VEIKK A15 Pro'
[Detect:Debug]  Searching for tablet 'VEIKK A15 V2'
[Detect:Debug]  Searching for tablet 'VEIKK A15'
[Detect:Debug]  Searching for tablet 'VEIKK A30 V2'
[Detect:Debug]  Searching for tablet 'VEIKK A30'
[Detect:Debug]  Searching for tablet 'VEIKK A50 (Variant 2)'
[Detect:Debug]  Searching for tablet 'VEIKK A50'
[Detect:Debug]  Searching for tablet 'VEIKK S640 V2'
[Detect:Debug]  Searching for tablet 'VEIKK S640'
[Detect:Debug]  Searching for tablet 'VEIKK VK1060'
[Detect:Debug]  Searching for tablet 'VEIKK VK1060PRO'
[Detect:Debug]  Searching for tablet 'VEIKK VK430 V2'
[Detect:Debug]  Searching for tablet 'VEIKK VK430'
[Detect:Debug]  Searching for tablet 'VEIKK VK640'
[Detect:Debug]  Searching for tablet 'VEIKK Voila (VO1060)'
[Detect:Debug]  Searching for tablet 'ViewSonic Woodpad PF0730'
[Detect:Debug]  Searching for tablet 'ViewSonic Woodpad PF1030'
[Detect:Debug]  Searching for tablet 'Wacom CTC-4110WL'
[Detect:Debug]  Searching for tablet 'Wacom CTC-6110WL'
[Detect:Debug]  Searching for tablet 'Wacom CTE-430'
[Detect:Debug]  Searching for tablet 'Wacom CTE-440'
[Detect:Debug]  Searching for tablet 'Wacom CTE-450'
[Detect:Debug]  Searching for tablet 'Wacom CTE-460'
[Detect:Debug]  Searching for tablet 'Wacom CTE-630'
[Detect:Debug]  Searching for tablet 'Wacom CTE-640'
[Detect:Debug]  Searching for tablet 'Wacom CTE-650'
[Detect:Debug]  Searching for tablet 'Wacom CTE-660'
[Detect:Debug]  Searching for tablet 'Wacom CTF-430'
[Detect:Debug]  Searching for tablet 'Wacom CTH-300'
[Detect:Debug]  Searching for tablet 'Wacom CTH-301'
[Detect:Debug]  Searching for tablet 'Wacom CTH-460'
[Detect:Debug]  Searching for tablet 'Wacom CTH-461'
[Detect:Debug]  Searching for tablet 'Wacom CTH-470'
[Detect:Debug]  Searching for tablet 'Wacom CTH-480'
[Detect:Debug]  Searching for tablet 'Wacom CTH-490'
[Detect:Debug]  Searching for tablet 'Wacom CTH-661'
[Detect:Debug]  Searching for tablet 'Wacom CTH-670'
[Detect:Debug]  Searching for tablet 'Wacom CTH-680'
[Detect:Debug]  Searching for tablet 'Wacom CTH-690'
[Detect:Debug]  Searching for tablet 'Wacom CTL-4100'
[Detect:Debug]  Searching for tablet 'Wacom CTL-4100WL'
[Detect:Debug]  Searching for tablet 'Wacom CTL-460'
[Detect:Debug]  Searching for tablet 'Wacom CTL-470'
[Detect:Debug]  Searching for tablet 'Wacom CTL-471'
[Detect:Debug]  Searching for tablet 'Wacom CTL-472'
[Detect:Debug]  Searching for tablet 'Wacom CTL-480'
[Detect:Debug]  Searching for tablet 'Wacom CTL-490'
[Detect:Debug]  Searching for tablet 'Wacom CTL-6100'
[Detect:Debug]  Searching for tablet 'Wacom CTL-6100WL'
[Detect:Debug]  Searching for tablet 'Wacom CTL-671'
[Detect:Debug]  Searching for tablet 'Wacom CTL-672'
[Detect:Debug]  Searching for tablet 'Wacom CTL-680'
[Detect:Debug]  Searching for tablet 'Wacom CTL-690'
[Detect:Debug]  Searching for tablet 'Wacom DTC-133'
[Detect:Debug]  Searching for tablet 'Wacom DTH-1320'
[Detect:Debug]  Searching for tablet 'Wacom Movink 13 (DTH-135)'
[Detect:Debug]  Searching for tablet 'Wacom Cintiq Pro 27 (DTH-271)'
[Detect:Debug]  Searching for tablet 'Wacom Cintiq 13HD (DTK-1300)'
[Detect:Debug]  Searching for tablet 'Wacom Cintiq 16 (DTK1660)'
[Detect:Debug]  Searching for tablet 'Wacom Cintiq 22HD (DTK-2200)'
[Detect:Debug]  Searching for tablet 'Wacom Cintiq 12WX (DTZ-1200W)'
[Detect:Debug]  Searching for tablet 'Wacom ET-0405-U'
[Detect:Debug]  Searching for tablet 'Wacom ET-0405A-U'
[Detect:Debug]  Searching for tablet 'Wacom FT-0405-U'
[Detect:Debug]  Searching for tablet 'Wacom GD-0405-U'
[Detect:Debug]  Searching for tablet 'Wacom GD-0608-U'
[Detect:Debug]  Searching for tablet 'Wacom GD-0912-U'
[Detect:Debug]  Searching for tablet 'Wacom GD-1212-U'
[Detect:Debug]  Searching for tablet 'Wacom GD-1218-U'
[Detect:Debug]  Searching for tablet 'Wacom MTE-450'
[Detect:Debug]  Searching for tablet 'Wacom PTH-450'
[Detect:Debug]  Searching for tablet 'Wacom PTH-451'
[Detect:Debug]  Searching for tablet 'Wacom PTH-460'
[Detect:Debug]  Searching for tablet 'Wacom PTH-650'
[Detect:Debug]  Searching for tablet 'Wacom PTH-651'
[Detect:Debug]  Searching for tablet 'Wacom PTH-660'
[Detect:Debug]  Searching for tablet 'Wacom PTH-850'
[Detect:Debug]  Searching for tablet 'Wacom PTH-851'
[Detect:Debug]  Searching for tablet 'Wacom PTH-860'
[Detect:Debug]  Searching for tablet 'Wacom PTK-1240'
[Detect:Debug]  Searching for tablet 'Wacom PTK-440'
[Detect:Debug]  Searching for tablet 'Wacom PTK-450'
[Detect:Debug]  Searching for tablet 'Wacom PTK-540WL'
[Detect:Debug]  Searching for tablet 'Wacom PTK-640'
[Detect:Debug]  Searching for tablet 'Wacom PTK-650'
[Detect:Debug]  Searching for tablet 'Wacom PTK-840'
[Detect:Debug]  Searching for tablet 'Wacom PTU-600U'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-1230'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-1231W'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-430'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-431W'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-630'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-631W'
[Detect:Debug]  Searching for tablet 'Wacom PTZ-930'
[Detect:Debug]  Searching for tablet 'Wacom XD-0405-U'
[Detect:Debug]  Searching for tablet 'Wacom XD-0608-U'
[Detect:Debug]  Searching for tablet 'Wacom XD-0912-U'
[Detect:Debug]  Searching for tablet 'Wacom XD-1212-U'
[Detect:Debug]  Searching for tablet 'Wacom XD-1218-U'
[Detect:Debug]  Searching for tablet 'Waltop Slim Tablet 5.8"'
[Detect:Debug]  Searching for tablet 'XenceLabs Pen Tablet Medium'
[Detect:Debug]  Searching for tablet 'XenceLabs Pen Tablet Small'
[Detect:Debug]  Searching for tablet 'XENX P1-640'
[Detect:Debug]  Searching for tablet 'XENX P3-1060'
[Detect:Debug]  Searching for tablet 'XENX X1-640'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 10 (2nd Gen)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 10S'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 12 (2nd Gen)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 12 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 12'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 13 (2nd Gen)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 13.3 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 13.3'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 15.6 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 15.6'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 16 (2nd Gen)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 16 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 16'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 22 (2nd Gen)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 22HD'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 24 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist 24'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist Pro 16 (Gen2)'
[Detect:Debug]  Searching for tablet 'XP-Pen Artist Pro 16TP'
[Detect:Debug]  Searching for tablet 'XP-Pen CT1060'
[Detect:Debug]  Searching for tablet 'XP-Pen CT430'
[Detect:Debug]  Searching for tablet 'XP-Pen CT640'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 01 V2 (Variant 2)'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 01 V2'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 01 V3'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 01'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 02'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco 03'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco L'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco M'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco mini4'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco mini7 V2'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco mini7'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco Pro LW Gen2'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco Pro Medium'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco Pro Small'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco Pro SW'
[Detect:Debug]  Searching for tablet 'XP-Pen Deco Pro XLW Gen2'
[Detect:Debug]  Searching for tablet 'XP-Pen Innovator 16'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 02'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 03 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 03'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 05 V3'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 06'
[Detect:Debug]  Searching for tablet 'XP-Pen Star 06C'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G430'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G430S V2'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G430S'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G540 Pro'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G540'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G640 (Variant 2)'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G640'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G640S'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G960'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G960S Plus'
[Detect:Debug]  Searching for tablet 'XP-Pen Star G960S'
[Detect:Info]   No tablets were detected.
помогите скачал win usb все равно не помогает планшет:graphic tablet
